src/api/graphql.js

```diff
@@ -0,0 +1,14 @@
+const gql = require('../utils/api-client')
+const { handleError } = require('../utils/api-error')
+
+const request = async ({ query, ...variables }) => {
```

Comment:
What version of node is `...spread` available in? Might need to check the examples `package.json` file just to make sure.

Reply:
Node version bumped to 8.3 (and updated examples README)